test/WebJobs.Script.Tests/Diagnostics/HostPerformanceManagerTests.cs

```diff
@@ -69,7 +70,60 @@ public void IsUnderHighLoad_Connections_ReturnsExpectedResults(int currentValue,
                 Connections = currentValue,
                 ConnectionLimit = 300
             };
-            Assert.Equal(expected, HostPerformanceManager.IsUnderHighLoad(counters));
+            Collection<string> exceededCounters = new Collection<string>();
+            Assert.Equal(expected, HostPerformanceManager.IsUnderHighLoad(counters, exceededCounters: exceededCounters));
+            if (expected)
+            {
+                Assert.Equal(1, exceededCounters.Count);
+                Assert.Equal("Connections", exceededCounters[0]);
+            }
+        }
+
+        [Theory]
+        [InlineData(0, false)]
+        [InlineData(10, false)]
+        [InlineData(480, false)]
+        [InlineData(481, true)]
+        [InlineData(500, true)]
+        [InlineData(600, true)]
+        [InlineData(610, true)]
+        public void IsUnderHighLoad_ActiveConnections_ReturnsExpectedResults(int currentValue, bool expected)
+        {
+            var counters = new ApplicationPerformanceCounters
+            {
+                ActiveConnections = currentValue,
+                ActiveConnectionLimit = 600
+            };
+            Collection<string> exceededCounters = new Collection<string>();
+            Assert.Equal(expected, HostPerformanceManager.IsUnderHighLoad(counters, exceededCounters: exceededCounters));
+            if (expected)
+            {
+                Assert.Equal(1, exceededCounters.Count);
+                Assert.Equal("ActiveConnections", exceededCounters[0]);
+            }
+        }
+
+        [Theory]
+        [InlineData(0, false)]
+        [InlineData(10, false)]
+        [InlineData(400, false)]
+        [InlineData(401, true)]
+        [InlineData(500, true)]
+        [InlineData(600, true)]
+        public void IsUnderHighLoad_RemoteDirMonitors_ReturnsExpectedResults(int currentValue, bool expected)
+        {
+            var counters = new ApplicationPerformanceCounters
+            {
+                RemoteDirMonitors = currentValue,
+                RemoteDirMonitorLimit = 500
```

Comment:
The limit of 500 can be enabled in 79 via a switch. Jaro will be experimenting to find the right value. For us, we just honor whatever limit we receive.